pages/thank-you.jsx

```diff
@@ -32,7 +32,7 @@ var ThankYou = React.createClass({
                 </h2>
               </div>
             </div>
-            {this.props.params && ['de', 'en-US'].indexOf(locale) !== -1 ? <Signup language={locale}/> : <Social language={locale}/> }
+            {this.props.params && ['de', 'en-US'].indexOf(locale) !== -1 ? <Signup locales={this.props.locales}/> : <Social language={locale}/> }
```

Comment:
Yeah, being consistent between pages and components.

this.props.locale as an array is what we have inside all pages passed in from client.jsx. The form mixin mostly works with pages, so it assumes this locales prop exists.

If we want the form mixin to work with components and not just pages, we need the locale to have the same signature.

Honestly, we could also make it so client.jsx passes in language as a single locale code and not be an array, but I figured locales being an array was needed? Whatever we decide on, it needs to be this.props.localething in all places so the mixin can use it too.